test/misc/multi_file_template_test.go

```diff
@@ -2773,6 +2775,31 @@ func TestMultiFileTemplate(t *testing.T) {
 			expectedOut: "hello, world!",
 		},
 
+		// Tests for extension-less imports using FormatFiles (FormatFS interface)
+		"import with 'for' - macro without file extension": {
+			sources: fstest.FormatFiles{
```

Comment:
It's not necessary to introduce `fastest.FormatFiles`, since files without an extension already default to the `Text` format.